ldap.go

```diff
@@ -35,31 +34,43 @@ func (x *ldapAuthenticator) Authenticate(identity, password string) (id string,
 	return
 }
 
-func (x *ldapAuthenticator) SetPassword(identity, password string) error {
+func (x *ldapAuthenticator) Close() {
+	if x.con != nil {
+		x.con.Close()
+		x.con = nil
+	}
+}
+
+type ldapUserStore struct {
+	//con *ldap.Conn
+	con *ldap.LDAPConnection
+}
+
+func (x *ldapUserStore) SetPassword(identity, password string) error {
 	return ErrUnsupported
 }
 
-func (x *ldapAuthenticator) ResetPasswordStart(identity string, expires time.Time) (string, error) {
+func (x *ldapUserStore) ResetPasswordStart(identity string, expires time.Time) (string, error) {
```

Comment:
I don't think we will ever support LDAP as a UserStore. For now, let's just implement LDAP as an Authenticator. This is the only real-world use case we've met so far.